python/triton/experimental/gluon/language/amd/cdna3/__init__.py

```diff
@@ -0,0 +1,28 @@
+from ..._core import builtin
+from triton._C.libtriton import ir
+
+__all__ = ["buffer_load_to_local"]
+
+
+@builtin
+def buffer_load_to_local(dest, ptr, offsets, mask=None, other=None, stride=None, cache_modifier="", _semantic=None):
```

Comment:
We don't use the "local" terminology in gluon
```suggestion
def buffer_load_to_shared(dest, ptr, offsets, mask=None, other=None, stride=None, cache_modifier="", _semantic=None):
```